src/lemlib/chassis/differential.cpp

```diff
@@ -143,17 +143,17 @@ void Differential::turnToHeading(float heading, int timeout, int maxSpeed) {
  * It also needs to decide what the chasePower should be. Usually this will be the value set in
  * the drivetrain struct, but it can be overridden by the user if needed.
  */
-void Differential::moveTo(float x, float y, float theta, int timeout, bool forwards, float chasePower, float lead,
+void Differential::moveTo(Length x, Length y, Angle theta, Time timeout, bool forwards, float chasePower, float lead,
                           int maxSpeed) {
     // if a movement is already running, wait until it is done
     if (movement != nullptr) waitUntilDone();
     // convert target theta to radians and standard form
-    Pose target = Pose(x, y, M_PI_2 - degToRad(theta));
+    Pose target = Pose(x, y, 90_deg - theta);
```

Comment:
```suggestion
    Pose target = Pose(x, y, M_PI_2_rad - theta);
```
change to radians for consistency